internal/clientconn/cursor/cursor.go

```diff
@@ -87,7 +86,7 @@ func newCursor(id int64, iter types.DocumentsIterator, params *NewParams, r *Reg
 		iter:      iter,
 		NewParams: params,
 		r:         r,
-		l:         r.l.With(zap.Int64("id", id), zap.Stringer("type", params.Type)),
+		l:         r.l.With(slog.Int64("id", id), slog.Any("type", params.Type)),
```

Comment:
`zap.Stringer(a, b)` should be replaced by `slog.String(a, b.String())`.

Please also fix that in a merged PR: https://github.com/FerretDB/FerretDB/pull/4457/files#r1681366889

Reply:
Thanks, replaced it to use String attribute.